ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -88,28 +88,30 @@ def _router_bgp_peer_groups(self) -> dict | None:
                 ]
             )
 
-        bgp_peer_groups = {}
+        bgp_peer_groups = []
         if peer_groups:
             for peer_group in peer_groups:
                 peer_group_name = peer_group.pop("name")
```

Comment:
This line removed the name but then you insert it again below. I suggest to just remove this pop and then append the peer_group directly.